andela/the_spinners-frontend
# #169817479 SASS setup

#### What does this PR do? 
`Set up SASS`
#### Description of Task to be completed?
```
Install and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client
```
#### How should this be manually tested?
- open terminal and run
- `git clone https://github.com/andela/the_spinners-frontend/`
- Open the project in VS code, then switch to this branch `git checkout ch-Setup-SASS-169817479`
- run `npm start` You will see `SASS TESTING` test changed to orange color, which is done using .scss instead of .css
#### What are the relevant pivotal tracker stories?
[#169817479](https://www.pivotaltracker.com/story/show/169817479)
#### Screenshots (if appropriate)
<img width="1299" alt="Screen Shot 2020-01-28 at 13 42 22" src="https://user-images.githubusercontent.com/58309608/73261096-0cb0b180-41d4-11ea-9cf8-caff26c107f1.png">
